mino/minows/key/disk.go

```diff
@@ -0,0 +1,64 @@
+package key
+
+import (
+	"crypto/rand"
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"go.dedis.ch/dela/core/store/kv"
+	"golang.org/x/xerrors"
+)
+
+// diskStorage provides persistent storage for private keys on disk.
+type diskStorage struct {
+	bucket []byte
+	db     kv.DB
+}
+
+func newDiskStore(db kv.DB) *diskStorage {
+	return &diskStorage{
+		bucket: []byte("keys"),
+		db:     db,
+	}
+}
+
+func (s *diskStorage) LoadOrCreate() (crypto.PrivKey, error) {
+	key := []byte("private_key")
+	var buffer []byte
+	err := s.db.Update(func(tx kv.WritableTx) error {
```

Comment:
Style: I would rather return the buffer here than doing the `copy`. Else it's not clear where the buffer will be filled.